src/mono/mono/mini/mini-arm64.c

```diff
@@ -2810,6 +2873,21 @@ mono_arch_allocate_vars (MonoCompile *cfg)
 			ins->inst_left = vtaddr;
 			break;
 		}
+		case ArgSwiftError: {
+			ins = mono_compile_create_var (cfg, mono_get_int_type (), OP_LOCAL);
+			ins->flags |= MONO_INST_VOLATILE;
```

Comment:
ins is already set above to cfg->args [i], there is no need to create a new one.

Reply:
You are right, both x64 and arm64 can use the existing ins.